fvm/evm/emulator/emulator.go

```diff
@@ -117,6 +117,8 @@ func (bl *BlockView) DirectCall(call *types.DirectCall) (*types.Result, error) {
 		return proc.mintTo(call, txHash)
 	case types.WithdrawCallSubType:
 		return proc.withdrawFrom(call, txHash)
+	case types.DryRunSubType:
+		return proc.dryRun(call)
```

Comment:
I guess we don't need this and Direct call subtype

Reply:
you are right, sorry forgot to remove